app/Resources/ShopsysFrameworkBundle/views/Admin/Layout/base.html.twig

```diff
@@ -0,0 +1,80 @@
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="robots" content="noindex, nofollow">
+
+        <link rel="stylesheet" type="text/css" href="{{ asset('assets/admin/styles/index_' ~ getCssVersion() ~ '.css') }}" media="screen, projection">
+        <link rel="stylesheet" type="text/css" href="{{ asset('bundles/bmatznerjqueryui/css/flick/jquery-ui.css') }}">
+        <!--[if IE 8 ]>
+            <link rel="stylesheet" type="text/css" href="{{ asset('assets/admin/styles/index_' ~ getCssVersion() ~ '-ie8.css') }}" media="screen, projection">
+        <![endif]-->
+
+        {# bootstrap/tooltip.js must be imported before bootstrap/popover.js #}
+        {{ importJavascripts([
+            'bundles/bmatznerjquery/js/jquery.min.js',
+            'bundles/bmatznerjquery/js/jquery-migrate.js',
+            'bundles/bmatznerjqueryui/js/minified/jquery-ui.min.js',
+            'common/components/filterAllNodes.js',
+            'common/components/keyCodes.js',
+            'common/components/escape.js',
+            'common/register.js',
+            'common/components/hybridTabs.js',
+            'common/components/tabs.js',
+            'common/components/timeout.js',
+            'common/components/toggleElement.js',
+            'common/plugins/BazingaJsTranslationBundle.translator.js',
+            'bundles/fpjsformvalidator/js/fp_js_validator.js',
+            'common/loaderOverlay.js',
+            'common/ajax.js',
+            'common/plugins/jquery.fix.clone.js',
+            'common/plugins/jquery.magnific-popup.js',
+            'common/plugins/jquery.ui.datepicker-cs.js',
+            'common/plugins/fastclick.js',
+            'common/validation/*.js',
+            'common/checkboxToggle.js',
+            'common/magnificPopup.custom.js',
+            'common/number.js',
+            'common/translator.js',
+            'common/bootstrap/tooltip.js',
+            'common/bootstrap/*.js',
+            'common/components/datePicker.js',
+            'common/components/tooltip.js',
+            'admin/plugins/*.js',
+            'admin/validation/form/*.js',
+            'admin/validation/*.js',
+            'admin/components/*.js',
+            'admin/*.js',
+            'custom_admin/*.js',
+            'frontend/plugins/chatbot.js',
+        ]) }}
+        {{ js_validator_config() }}
+        {{ init_js_validation() }}
+
+        <script type="text/javascript">
+            global = {
+                locale: '{{ app.request.locale }}'
+            };
+        </script>
+
+        <title>{{ 'Administration'|trans }} {% block title %}Shopsys Framework{% endblock %}</title>
+    </head>
+
+    <body>
+        {% block html_body %}{% endblock %}
+
+        {% if app.environment == 'dev' %}
+            <script async src="//localhost:35729/livereload.js"></script>
+        {% endif %}
+
+        <style type="text/css">
```

Comment:
is this because of the fixed bar we have on many pages in the admin?

Reply:
Exactly